spikeinterface/core/waveform_extractor.py

```diff
@@ -449,12 +449,61 @@ def _waveform_extractor_chunk(segment_index, start_frame, end_frame, worker_ctx)
 
 def extract_waveforms(recording, sorting, folder,
                       load_if_exists=False,
-                      ms_before=3., ms_after=4., max_spikes_per_unit=500, dtype=None, **job_kwargs):
+                      ms_before=3., ms_after=4.,
+                      max_spikes_per_unit=500,
+                      overwrite=False,
```

Comment:
OK for overwrite but we need a way to check that load_if_exists and overwrite are mutually exclusive